src/compiler/sys.ts

```diff
@@ -340,11 +340,20 @@ namespace ts {
                     }
 
                     function fileChanged(curr: any, prev: any) {

```

Comment:
It was this way before, but it would be nice to remove `any` and include a type declaration saying what you expect this to be.